### ISSUE TITLE
Hello, I am trying to learn your code, but this operation has been reported as follows, could you please tell me how to solve it?

### ISSUE THREAD

**Issue by yafenhe666** [KeiLongW/battery-state-estimation#7]
![error](https://user-images.githubusercontent.com/95292669/199717022-64b1bb86-d8e8-4924-a60b-c74c283c4675.PNG)

**Comment by KeiLongW** [comment]
it seems you are trying to fit 3D data to a layer that accepts 2D input. 
If the layer before `dense_3` is an LSTM layer, maybe you need to set `return_sequences=False`?

Anyway, it is better to show your code when asking this type of question.

**Comment by yafenhe666** [comment]
Thank you very much for your reply and suggestion. I'll give it a try next. Let me ask you again, the loss function used in your model is ‘huber’, but when I was running it, it indicated that it did not have this function, and then I changed it to  ‘logcosh’ function. Is it a cause of the above problem?
![1](https://user-images.githubusercontent.com/95292669/199729397-a7fc8b8d-1d7c-4fc5-a45f-f910acb7db50.PNG)

**Comment by KeiLongW** [comment]
typo? it is `huber` not `humber`.

**Comment by yafenhe666** [comment]
oh，sorry，i use huber ，and it is also has an error ：“Unknown loss function:huber”
![2](https://user-images.githubusercontent.com/95292669/199730829-0cfde9a9-a4c7-4364-a466-70ae9aa4b890.PNG)

**Comment by KeiLongW** [comment]
what is your TensorFlow version? please try to follow the requirements in `environment.yml`.

**Comment by yafenhe666** [comment]
I run it， TensorFlow version is 2.1.0，python  version is 3.6， Keras version is 2.3.1.
And  I  come from ChengDu，SiChuan，China。I see you are come from Macao，China ，It's a pleasure to meet you.

**Comment by KeiLongW** [comment]
According to the Doc: https://www.tensorflow.org/versions/r2.1/api_docs/python/tf/keras/losses
No huber loss Function in `tf.keras.losses` for version 2.1, so upgrading TensorFlow should solve the issue.
BTW, nice to meet you :)

**Comment by yafenhe666** [comment]
Oh，I will deal with it your way and express my heartfelt thanks again for your answer.
